ArchiSteamFarm/www/js/app.js

```diff
@@ -1035,14 +1035,15 @@ $(function () {
             async: false,
             success: function (data) {
                 var obj = data['Result'];
+                const languageRegex = new RegExp("strings.[a-z]{2}-[A-Z]{2}.json|strings.json");
```

Comment:
I'm not regex expert but wouldn't `strings(.[a-z]{2}-[A-Z]{2})?.json` be better?

Reply:
Exactly what I wanted to do initaly, but my cheatsheet had no explanation for `()?`